src/REVDeployer.sol

```diff
@@ -722,6 +739,19 @@ contract REVDeployer is ERC2771Context, IREVDeployer, IJBRulesetDataHook, IJBCas
             revert REVDeployer_RulesetDoesNotAllowDeployingSuckers();
         }
 
+        // Keep a reference to the hashed encoded configuration of the revnet.
+        bytes32 storedHashedEncodedConfiguration = hashedEncodedConfigurationOf[revnetId];
+
+        // Keep a reference to the proposed hashed encoded configuration.
+        bytes32 proposedHashedEncodedConfiguration = keccak256(encodedConfiguration);
+
+        // If the proposed configuration doesn't match the stored configuration, revert.
+        if (storedHashedEncodedConfiguration != proposedHashedEncodedConfiguration) {
```

Comment:
I think we can just use the stored hash, afaik don't need the full `encodedConfiguration`

Reply:
https://github.com/rev-net/revnet-core/pull/63